wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,74 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+A user may get banned if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Sharing their account with other users.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What is considered cheating?
+The rules say that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player.
 
-This is a one-strike rule; cheaters will be **permanently banned.**
+This is a one-strike rule; cheaters will be *permanently banned*.
```

Comment:
; → : 
I've seen people get unbanned after some months and appeals. Maybe this section needs rewording, too? Also conflicts with the "What can I do if I've broken the rules?" section, as it says the ban *may* not be permanent.

Reply:
I'll consult the higher-ups about this.

Reply:
What "permanent" means is that it's not a time-based ban, like in many other games. Also, the "what can i do..." section specifically states that no action will be taken.

Change for this:
This is a one-strike rule. Cheaters will be banned on their first offence and for an indefinite amount of time. The team may decide to lift their ban if they successfully appeal.